cmd/oras/internal/display/status/interface.go

```diff
@@ -61,3 +63,24 @@ type CopyHandler interface {
 	PostCopy(ctx context.Context, desc ocispec.Descriptor) error
 	OnMounted(ctx context.Context, desc ocispec.Descriptor) error
 }
+
+// ManifestIndexCreateHandler handles status output for manifest index create command.
+type ManifestIndexCreateHandler interface {
+	OnSourceManifestFetching(source string) error
+	OnSourceManifestFetched(source string) error
+	OnIndexPacked(shortDigest string) error
+	OnIndexPushed(path string) error
+}
+
+// ManifestIndexUpdateHandler handles status output for manifest index update command.
+type ManifestIndexUpdateHandler interface {
+	OnIndexFetching(indexRef string) error
+	OnIndexFetched(indexRef string, digest digest.Digest) error
+	OnManifestFetching(manifestRef string) error
+	OnManifestFetched(manifestRef string, digest digest.Digest) error
+	OnManifestRemoved(digest digest.Digest) error
+	OnManifestAdded(manifestRef string, digest digest.Digest) error
+	OnIndexMerged(indexRef string, digest digest.Digest) error
```

Comment:
Should be moved to metadata handler

Reply:
Moved Packed, Updated, Pushed, Added, Removed, Merged to metadata handlers.